### PR TITLE
Revert "Kill DOMRectList"

### PR DESCRIPTION
**Pull request by zcorpan**
See https://github.com/w3c/csswg-drafts/issues/1479

Tests: https://github.com/w3c/web-platform-tests/issues/6299

### CODE REVIEW

**Comment by foolip** [comment]
Is this blocked only on review? @fserb FYI

**Comment by fserb** [comment]
Friendly ping on this. I've just updated Chrome Canary to this, so it would be nice to have a response on it soon.

**Comment by jpmedley** [comment]
Pinging on this again. I need to mention @fserb 's update in a blog post about Chrome 61. I'm blocked if this hasn't landed.

**Comment by foolip** [comment]
I'm pretty sure @zcorpan isn't working this week. Who else has the power to merge things in this repo?

**Comment by foolip** [comment]
The most recent commit in the directory by anyone other than @zcorpan is https://github.com/w3c/fxtf-drafts/commit/760283dbb59ca1722965a9ba53cafc857a5d02ef. @tabatkins, if this is reviewed, can you merge it?

File: geometry/Overview.bs [hunk 1]
```diff
@@ -372,6 +372,29 @@ steps:
 </div>
 
 
+<h2 id=DOMRectList>The DOMRectList interface</h2>
+
+<pre class=idl>
+[NoInterfaceObject,
+ ArrayClass]
```
Comment (foolip) [inline geometry/Overview.bs]: Should be [LegacyArrayClass] per https://heycam.github.io/webidl/#LegacyArrayClass.

**Comment by tabatkins** [comment]
Done 👍

**Comment by foolip** [comment]
Thanks!

**Pull request merged**